subsys/bluetooth/audio/has.c

```diff
@@ -996,6 +1062,10 @@ int bt_has_preset_register(const struct bt_has_preset_register_param *param)
 		return -ENOMEM;
 	}
 
+#if defined(CONFIG_BT_HAS_FEATURES_NOTIFIABLE)
+	k_work_submit(&features_work);
+#endif /* CONFIG_BT_HAS_FEATURES_NOTIFIABLE */
```

Comment:
As far as I can tell, this function does not change the `features` value, so why are we submitting work to notify the features?

Reply:
You are right, this is not where it was supposed to be.